src/Workspaces/Core/Portable/Workspace/Solution/TextDocumentStates.cs

```diff
@@ -137,8 +147,10 @@ public ImmutableArray<TValue> SelectAsArray<TValue>(Func<TState, TValue> selecto
     }
 
     public TextDocumentStates<TState> AddRange(ImmutableArray<TState> states)
+        // TODO: we could more efficiently generate the new filePathToDocumentIds here.
```

Comment:
>// TODO: we could more efficiently generate the new filePathToDocumentIds here. [](http://example.com/codeflow?start=8&length=79)

Feels like we could remove these TODOs as it's questionable whether that work would even be valuable.